runtime_test.go

```diff
@@ -122,11 +128,75 @@ func (s *runtimeSuite) TestRuntime(c *C) {
 	}
 	ok, val = failpoint.Eval("runtime-test-6")
 	c.Assert(ok, IsFalse)
+	c.Assert(val, IsNil)
 
 	ok, val = failpoint.Eval("failpoint-env1")
 	c.Assert(ok, IsTrue)
 	c.Assert(val.(int), Equals, 10)
 	ok, val = failpoint.Eval("failpoint-env2")
 	c.Assert(ok, IsTrue)
 	c.Assert(val.(bool), IsTrue)
+
+	// Tests for sleep
+	ch = make(chan struct{})
+	go func() {
+		time.Sleep(time.Second)
+		err := failpoint.Disable("gofail/test-sleep")
+		c.Assert(err, IsNil)
+		close(ch)
```

Comment:
Change this to a `defer` at the beginning, so even if the assertion failed it the `<-ch` below won't block the process forever.